docs/source/getting-started/running-notebooks/index.rst

```diff
@@ -0,0 +1,139 @@
+How to Run Tutorials
+====================
+
+
+Throughout the MLflow documentation, you'll find a collection of notebook-based tutorials that will help to provide a primer on learning concepts of MLflow. 
+From the very `first getting started tutorials <../intro-quickstart/index.html>`_ to in-depth use-case driven examples of using MLflow with popular ML frameworks 
+`like transformers <../../llms/transformers/index.html>`_, these tutorials 
+typically integrate with the MLflow Tracking Server (to learn more about it, see `the guide here <../../tracking.html>`_ and view the 
+`5 minute quickstart tutorial here <https://www.mlflow.org/docs/latest/getting-started/tracking-server-overview/index.html>`_) to record information about 
+experiments, runs, and the artifacts that are produced by your machine learning code. 
```

Comment:
Do we need this section to explain what tutorials are? Maybe we could just add links to some examples in below `This brief guide will walk you through some options that you have to run tutorials (such as ...) and have ...`

Reply:
Good callout. I'm removing that top paragraph